Makefile

```diff
@@ -471,18 +471,20 @@ docker-start-bash:
 docker-start-dev:
 	docker run --rm \
 	-v ${CURRENT_DIR}:/mnt/local_unstructued \
-	-ti ${DOCKER_IMAGE}
+	-ti --entrypoint /bin/bash ${DOCKER_IMAGE}
 
 .PHONY: docker-test
 docker-test:
 	docker run --rm \
-	-v ${CURRENT_DIR}/test_unstructured:/home/notebook-user/test_unstructured \
-	-v ${CURRENT_DIR}/test_unstructured_ingest:/home/notebook-user/test_unstructured_ingest \
-	$(if $(wildcard uns_test_env_file),--env-file uns_test_env_file,) \
-	$(DOCKER_IMAGE) \
-	bash -c "CI=$(CI) \
-	UNSTRUCTURED_INCLUDE_DEBUG_METADATA=$(UNSTRUCTURED_INCLUDE_DEBUG_METADATA) \
-	pytest -m 'not chipper' $(if $(TEST_FILE),$(TEST_FILE),test_unstructured)"
+		$(if $(wildcard uns_test_env_file),--env-file uns_test_env_file,) \
+		$(DOCKER_IMAGE) \
+		/home/nonroot/.local/bin/pytest -m 'not chipper' test_unstructured
```

Comment:
@tabossert - And another mystery here: I added `/home/nonroot/.local/bin` to `PATH` in the `Dockerfile` but it didn't find it here without the explicit path. Feels like its related to the other comment above but can't figure out why.